lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.h

```diff
@@ -0,0 +1,49 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+#ifdef ENABLE_JS_BUILTINS
+namespace Js
+{
+    class JsBuiltInEngineInterfaceExtensionObject : public EngineExtensionObjectBase
+    {
+    public:
+        JsBuiltInEngineInterfaceExtensionObject(ScriptContext* scriptContext);
+        void Initialize();
+        void InitializePrototypes(ScriptContext * scriptContext);
+        void InjectJsBuiltInLibraryCode(ScriptContext * scriptContext);
+
+        static bool __cdecl InitializeJsBuiltInNativeInterfaces(DynamicObject* intlNativeInterfaces, DeferredTypeHandlerBase * typeHandler, DeferredInitializeMode mode);
+
+#if DBG
+        void DumpByteCode() override;
+#endif
+
+        class EntryInfo
+        {
+        public:
+            static NoProfileFunctionInfo JsBuiltIn_RegisterChakraLibraryFunction;
+            static NoProfileFunctionInfo JsBuiltIn_RegisterFunction;
+            static NoProfileFunctionInfo JsBuiltIn_Internal_GetArrayLengthFunction;
```

Comment:
GetLength instead of GetArrayLength (as we are fetching the 'length' property).

Reply:
Done.